Documentation/Reference.md

```diff
@@ -61,51 +61,117 @@ Where `foo` is an executable and `bar.a` a static library.
 
 ## Package Manifest File Format Reference
 
-Instructions for how to build a package are provided by a manifest file, called `Package.swift`. You can customize this file to declare build targets or dependencies, include or exclude source files, and specify build configurations for the module or individual files.
+Instructions for how to build a package are provided by the `Package.swift` manifest file. `Package.swift` is a Swift file that declaratively configures a Package using types defined in the `PackageDescription` module. It currently has following properties:
 
-Here's an example of a `Package.swift` file:
+### name
+
+Defines the name of the package and is the minimal requirement for a manifest to be valid. When the sources are located directly under `Sources/` directory, there is only one module and the module name is same as the package name.
+
+```swift
+import PackageDescription
+
+let package = Package(
+    name: "FooBar"
+)
+```
+
+### targets
+
+Targets property is used when you have more than one module in your package and want to declare inter-dependency between them.
```

Comment:
This should be "interdependencies", but I would just say "a dependency between them".